### PR TITLE
tox.ini: Enable Python 3.13

### PR DESCRIPTION
**Pull request by bkeryan**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nidaqmx-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Enable running system tests with Python 3.13.

### Why should this Pull Request be merged?

Finish fixing #644 

### What testing has been done?

None

### CODE REVIEW

**Comment by bkeryan** [comment]
The py64 runner has been updated but the py32 runner has not.

File: tox.ini [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 [tox]
 isolated_build = true
-envlist = clean, py{38,39,310,311,312}-base, py{38,39,310,311,312}-grpc, py39-base-nicaiu, py39-base-nicai_utf8, report, docs
+envlist = clean, py{38,39,310,311,312,313}-base, py{38,39,310,311,312,313}-grpc, py39-base-nicaiu, py39-base-nicai_utf8, report, docs
```
Comment (zhindes) [inline tox.ini:8]: FYI - Python 3.8 is end of life, which per [NI Driver ADE Support Policy](https://dev.azure.com/ni/DevCentral/_git/Policy?path=/product_and_technology/Driver-ADE-Support.md&_a=preview) means we can remove it. We shouldn't do it here, though. I'll create a new issue for this.

**Review by zhindes — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (zhindes) [reply tox.ini:8]: #647

Comment (bkeryan) [reply tox.ini:8]: Yeah, I think we should cherry-pick the 3.13 fix into releases/1.0 and remove 3.8 in 1.1.

**Comment by bkeryan** [comment]
I double-checked the output to make sure it's actually running 3.13.

py32: https://github.com/ni/nidaqmx-python/actions/runs/11376831246/job/31650061261?pr=646
```
py313-base: skipped because could not find python interpreter with spec(s): py313
py313-base: SKIP ⚠ in 2.42 seconds
```
This is due to the fact that creating an updated py32 image failed.

py64: https://github.com/ni/nidaqmx-python/actions/runs/11376831246/job/31650061486?pr=646
```
py313-grpc: commands[0]> poetry run python --version
Python 3.13.0
```

py64 works and py32 fails gracefully, so I guess this PR doesn't have to wait for the new py32 image.

**Comment by bkeryan** [comment]
```
PS D:\dev\nidaqmx-python> poetry run pytest -v
=================================================== test session starts ===================================================
platform win32 -- Python 3.13.0, pytest-8.3.3, pluggy-1.5.0 -- D:\dev\nidaqmx-python\.venv\Scripts\python.exe
...
================================================= short test summary info =================================================
FAILED tests/component/system/test_device_properties.py::test___ext_cal_last_date_and_time___no_errors[library_init_kwargs] - assert 2009 > 2009
FAILED tests/component/system/test_device_properties.py::test___ext_cal_last_date_and_time___no_errors[grpc_init_kwargs] - assert 2009 > 2009
=========================== 2 failed, 2286 passed, 48 skipped, 11 xfailed in 212.89s (0:03:32) ============================
```

**Pull request merged**